src/index.ts

```diff
@@ -374,7 +383,17 @@ class SOOSSCAAnalysis {
       }
 
       if (allUploadsFailed) {
-        throw new Error("Error uploading manifests.");
+        await soosAnalysisApiClient.updateScanStatus({
+          clientId: this.args.clientId,
+          projectHash,
+          branchHash,
+          scanType: ScanType.SCA,
+          scanId: analysisId,
+          status: ScanStatus.Incomplete,
+          message: `Error uploading manifests.`,
```

Comment:
you don't need back tics for this message - make a local const and share with the logger too